kafka_utils/kafka_cluster_manager/cmds/replace.py

```diff
@@ -50,8 +52,8 @@ def build_subparser(self, subparsers):
         subparser.add_argument(
             '--dest-broker',
             type=int,
-            required=True,
-            help='Broker id of destination broker.',
+            default=None,
+            help='Broker id of destination broker. If empty, will shrink the replica set of topics',
```

Comment:
What will happen if this is called multiple times without a dest-broker such that the replicas becomes 0?